.github/workflows/release-doc.yml

```
@@ -17,7 +17,7 @@ jobs:
         env:
           release_tag: ${{ github.ref }}
       - name: Convert mkdocs
-        shell: bash
+        shell: bash --noprofile --norc -e {0}
```

```suggestion
        shell: bash --noprofile --norc -ex {0}
```
Maybe adding -x (aka `xtrace`) will help troubleshoot?

---

The `-o pipefail` made the pipeline fail on `grep` if the pattern was not found